main.go

```diff
@@ -6,12 +6,17 @@ import (
 	"strconv"
 	"time"
 
+	log "github.com/sirupsen/logrus"
+
 	"github.com/guoger/stupid/infra"
 )
 
 func main() {
+	log.SetOutput(os.Stdout)
+	log.SetLevel(log.WarnLevel)
+	logger := log.New()
 	if len(os.Args) != 3 {
-		fmt.Printf("Usage: stupid config.json 500\n")
+		logger.Panicln("Usage: stupid config.json 500")
```

Comment:
We should keep this to use `fmt.Println`, since it's not exactly a log, but user instruction